frontend/src/v5/ui/routes/viewer/groups/groupLists.styles.ts

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
The name of the file is not matching the others'
- current file groupList**S**.styles.ts (note the `s` after list)
- other files groupList.*

Also, if the files are called groupList, could the folder also be named like that?

Reply:
I was planni g to use that directory for near future groups components. But agree with the rest .